modules/post/multi/escalate/aws_create_iam_user.rb

```diff
@@ -0,0 +1,94 @@
+require 'msf/core'
+require 'metasploit/framework/aws/client'
+
+class MetasploitModule < Msf::Post
+
+  include Metasploit::Framework::Aws::Client
+
+  def initialize(info={})
+    super(update_info(info,
+      'Name'           => "Create an AWS IAM User",
+      'Description'    => %q{
+        This module will attempt to create an AWS (Amazon Web Services) IAM
+        (Identity and Access Management) user with Admin privileges.
+      },
+      'License'        => MSF_LICENSE,
+      'Platform'      => %w(unix),
+      'SessionTypes'  => %w(shell meterpreter),
+      'Author'         => ['Javier Godinez <godinezj[at]gmail.com>']
```

Comment:
Will be referencing https://github.com/devsecops/bootcamp/raw/master/Week-6/slides/june-DSO-bootcamp-week-six-lesson-three.pdf